src/Application/UI/Presenter.php

```diff
@@ -654,6 +661,7 @@ final public function getLastCreatedRequestFlag(string $flag): bool
 	/**
 	 * Conditional redirect to canonicalized URI.
 	 * @throws Nette\Application\AbortException
+	 * @return never|void
```

Comment:
@JanTvrdik Can be never combined?

Reply:
It can't be https://github.com/php/php-src/blob/master/Zend/tests/type_declarations/union_types/never_with_class.phpt

Reply:
Okay, but how do we solve this? The method often returns `never`, but sometimes `null`. Neither typehint is correct.

Reply:
> In type theory noreturn would be called a “bottom” type. That means it's effectively a subtype of every other type in PHP’s type system, including void.
https://wiki.php.net/rfc/noreturn_type#variance

Void is correct.

Reply:
Ok, reverted back to `void`.